pkg/disk/lsblk_flatten.go

```diff
@@ -0,0 +1,116 @@
+package disk
+
+import (
+	"io"
+	"sort"
+	"strings"
+
+	"github.com/dustin/go-humanize"
+	"github.com/olekukonko/tablewriter"
+)
+
+type FlattenedBlockDevices []FlattenedBlockDevice
+
+// FlattenedBlockDevice represents the flattened output of lsblk command for a device,
+// meaning the child device is included as its own entry in the device list.
+type FlattenedBlockDevice struct {
+	Name       string `json:"name,omitempty"`
+	Type       string `json:"type,omitempty"`
+	Size       uint64 `json:"size,omitempty"`
+	Rota       bool   `json:"rota,omitempty"`
+	Serial     string `json:"serial,omitempty"`
+	WWN        string `json:"wwn,omitempty"`
+	Vendor     string `json:"vendor,omitempty"`
+	Model      string `json:"model,omitempty"`
+	Rev        string `json:"rev,omitempty"`
+	MountPoint string `json:"mountpoint,omitempty"`
+	FSType     string `json:"fstype,omitempty"`
+	PartUUID   string `json:"partuuid,omitempty"`
+
+	Parents  []string `json:"parents,omitempty"`
+	Children []string `json:"children,omitempty"`
+}
+
+func (blks BlockDevices) Flatten() FlattenedBlockDevices {
```

Comment:
```suggestion
func (blks *BlockDevices) Flatten() FlattenedBlockDevices {
```

Reply:
`BlockDevices` is already reference (slice)

Reply:
yea... loll..